Feature/add presentations (#108)
vuejs/events: src/presentations/README.md

---
sidebar: auto
---

# Introduction

## Introduction

> Vue.js Events is an effort to centralize all public presentation about Vue.js and associated library and framework. It allow to quiclky find a presentation by theme and year. It also allow event organizer to reuse presentation to share knowledge at local area without recreating presentation each time.

Reviewer: ```suggestion
This is an effort to centralize all public presentation about Vue.js and associated library and framework. It allow to find a presentation by theme and year. It also allow speakers to find presentations that are open to reuse in order to share knowledge at local area without recreating presentation each time.
```

> Vue.js Events is an effort to centralize all public presentation about Vue.js and associated library and framework. It allow to quiclky find a presentation by theme and year. It also allow event organizer to reuse presentation to share knowledge at local area without recreating presentation each time.

Reviewer 2: Changing the pluralization, tense, and general language to match a prose more similar to the average native English speaker, while conveying the same ideas of the original passage. Specifically changing usage of "allow" where it does not actually grant additional permission.

```suggestion
This is an effort to centralize all public presentations about Vue.js and associated libraries/frameworks. It catalogs presentations by theme and year, and can be used to help speakers find presentations that are open for reuse. So that knowledge can be shared at local events without having to recreate each presentation.
```

> // TODO more introduction ? Wording ?

Reviewer: Can delete this for now

---
sidebar: auto
---

# Presentations

## Introduction

This is an effort to centralize all public presentations about Vue.js and associated libraries/frameworks. It catalogs presentations by theme and year, and can be used to help speakers find presentations that are open for reuse. So that knowledge can be shared at local events without having to recreate each presentation.

To submit your go to the end of the page or [click here](/presentations/#submit)

## Discover

<PresentationList />

## Submit

> This page is only interresting if YOU share your presentation with the community.

Author: Wording here

Reviewer: If you'd like to share your presentation with the community, all you need to do is create a pull request 
```suggestion
Sharing your presentation with the community is entirely voluntary.
```

> Adding your is really simple. You just have to create a pull request with a json file containing all the data about the presentation you want to share.

Reviewer: ```suggestion
To add your presentation, you need to create a pull request with a JSON file containing all the data about the presentation you want to share.
```
